docs/source/index.rst

```diff
@@ -22,19 +22,17 @@ You can get instructions on how to download and install Devito
 :doc:`here </download>`.
 
 To learn how to use Devito, check our :doc:`tutorials and examples </tutorials>`.
+Here you will also find documentation about the inner workings of the compiler.
 
 You can find the API Reference, which includes detailed explanations and
 inline examples, :doc:`here </userapi>`.
 
-If you want to know more about how the Devito compiler works,
-have a look at this :doc:`material </compiler>`.
-
 .. title:: The Devito project
 
 .. toctree::
    :hidden:
 
    Download <download>
-   Tutorials <tutorials>
+   Tutorials <https://github.com/devitocodes/devito/tree/master/examples>
```

Comment:
nbviewer is a lot nicer for tutorials so I prefer the previous way. May need some reorga to make it easier to link though.

Reply:
oh I see, I had missed that. I was just trying to drop redundant places -- we have a nice description of what our tutorials are in `examples/README.md` and we had another one, incomplete and obsolete (!),  in `tutorials.rst`, which therefore I'm keen to drop, otherwise, it's just maintenance effort. 

How about we add a link to nbviewer in `examples/README.md` ?

Reply:
I think that'd be fine

Reply:
done now, have a look